docs/providers/terraform/aws.mdx

```diff
@@ -118,6 +118,10 @@ config:
       # set a timeout of 15 seconds
       # See lambda timeout values here:
       # https://docs.aws.amazon.com/lambda/latest/dg/configuration-function-common.html#configuration-timeout-console
+      ephemeral-storage: 512
+      # set a provisioned concurrency value
+      # For info on provisioned concurrency for AWS Lambda see:
+      # https://docs.aws.amazon.com/lambda/latest/dg/configuration-concurrency.html
       timeout: 15
       # set a provisioned concurrency value
       # For info on provisioned concurrency for AWS Lambda see:
```

Comment:
Can you update the comments here, it looks like there weren't quite right before this PR so it's a good chance to clean them up